packages/web-api/src/types/request/bots.ts

```diff
@@ -0,0 +1,7 @@
+import type { OptionalTeamAssignable, TokenOverridable } from './common';
+
+// https://api.slack.com/methods/bots.info
+export interface BotsInfoArguments extends TokenOverridable, OptionalTeamAssignable {
+  /** @description Bot user ID to retrieve information about. */
```

Comment:
Bot user ID can be misleading because many of our incoming payloads can contain bot_id (B...) and bot_user_id (U/W...)
```suggestion
  /** @description Bot ID, which starts with 'B', to retrieve information about. */
```